app/models/shipit/task.rb

```diff
@@ -23,6 +23,8 @@ class Task < ActiveRecord::Base
     belongs_to :until_commit, class_name: 'Commit', required: false
     belongs_to :since_commit, class_name: 'Commit', required: false
 
+    belongs_to :rollback_once_aborted_to, class_name: 'Task', optional: true
```

Comment:
`rollback_once_aborted_to` reads a bit weird but I can't think of anything better.

Reply:
yeah it does, but at least its kinda descriptive 😛